src/instructlab/sdg/pipeline.py

```diff
@@ -39,12 +39,6 @@ def generate(self, dataset) -> Dataset:
             drop_duplicates_cols = block_prop.get("drop_duplicates", False)
             block = block_type(**block_config)
 
-            if block_type == IterBlock:
```

Comment:
e2e is failing with:

```
  File "/home/runner/work/sdg/sdg/venv/lib/python3.11/site-packages/instructlab/sdg/pipeline.py", line 6, in <module>
    from .iterblock import IterBlock
ModuleNotFoundError: No module named 'instructlab.sdg.iterblock'
```

see above warnings

```
E0401: Unable to import 'instructlab.sdg.iterblock' (import-error)
W0611: Unused IterBlock imported from iterblock (unused-import)
```

Reply:
Ah yeah, the pipeline file changes didn't get pushed, sigh!